packages/team-management/src/components/TeamsView/index.tsx

```diff
@@ -176,9 +178,16 @@ export const TeamsView: React.FC<TeamsViewTypes> = (props: TeamsViewTypes) => {
     const response: Organization[] = await teamsService.list(filterParams as Dictionary);
     dispatch(removeOrganizationsAction());
     dispatch(fetchOrganizationsAction(response));
+    setApiError(true);
     return response;
   }
 
+  const onError = () => {
+    setApiError(true);
+  };
+
+  if (apiError) return <BrokenPage />;
+
```

Comment:
This does not look right, to start with: seems like apiError is set to true unconditionally